packages/fx-core/templates/plugins/resource/sql/bicep/input_param.template.bicep

```diff
@@ -0,0 +1,8 @@
+
+param azureSql_aadAdmin string
+param azureSql_aadAdminObjectId string
+param azureSql_tenantId string
+param azureSql_admin string
+param azureSql_adminPassword string
```

Comment:
should add `@secure()` decorator for credentials

Reply:
Sure